### PR TITLE
Refractor

### PR DESCRIPTION
**Pull request by CFin86**
finished!

### CODE REVIEW

File: RefactorKata/Program.cs [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
-            //This is intentionally bad : (  Let's Refactor!
-            System.Data.SqlClient.SqlConnection Conn = new System.Data.SqlClient.SqlConnection("Server=.;Database=myDataBase;User Id=myUsername;Password = myPassword;");
-
-            System.Data.SqlClient.SqlCommand cmd = Conn.CreateCommand();
-            cmd.CommandText = "select * from Products";
-            /*
-             * cmd.CommandText = "Select * from Invoices";
-             */
-            System.Data.SqlClient.SqlDataReader reader = cmd.ExecuteReader();
             List<Product> products = new List<Product>();
```
[the remark this answers is not in the document]
Comment (codytwinton) [reply RefactorKata/Program.cs]: @CFin86 Don't forget the `var` keyword here. Like: `var products = new List<Product>();`

**Comment by CFin86** [comment]
After reviewing your requested changes, I realized that when I did my commit last night I had not saved my local files before committing and pushing. I'm used to VS Code that has an autosave feature, haha

File: RefactorKata/Program.cs [hunk 2]
```diff
@@ -1,41 +1,38 @@
...
+                                                "Password = myPassword;"))
             {
-                var prod = new Product();
-                prod.name = reader["Name"].ToString();
-                products.Add(prod);
+                var cmd = conn.CreateCommand();
+                cmd.CommandText = "select * from Products";
+
+                var reader = cmd.ExecuteReader();
+
```
Comment (codytwinton) [inline RefactorKata/Program.cs]: Unnecessary whitespace.
